src/Common/src/Interop/Unix/System.Native/Interop.Dup2.cs

```diff
@@ -0,0 +1,15 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Runtime.InteropServices;
+
+internal static partial class Interop
+{
+    internal static partial class Sys
+    {
+        [DllImport(Libraries.SystemNative, EntryPoint = "SystemNative_Dup2", SetLastError = true)]
+        internal static extern Error Dup2(SafeHandle oldfd, SafeHandle newfd);
+    }
+}
```

Comment:
Is all of this dup2 stuff that was added still needed, or can it be undone?  Seems like with the revised approach, it's no longer necessary.